.changeset/yellow-tables-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jpmorganchase/uitk-core": minor
+---
+
+Removed polymorphic types from Button
```

Comment:
Can we expand on this please. Only saying the types are being removed is a bit misleading.